sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
+It allows capture of features that describe common concepts across input
+modalities. Generative graphical models like restricted Boltzmann
+machines (RBM) and their stacked versions, deep Boltzmann machines (DBM), and
+deep belief networks (DBN), demonstrate successful extraction of
+more informative features for one modality
+(images or video) when jointly learnt with other modalities
+(audio or text) [@tag:Ngiam2011].
+Jha et al. [@tag:Jha2017_integrative_models] showed that an integrated training
+approach delivers better performance compared to individual networks. They
+compared a number of feed-forward architectures trained on RNA-Seq data
+with and without an additional set of CLIP-seq, knockdown, and over-expression
+based input features. Results showed that the integrative deep model
+generalized well for combined data, offering large performance improvement
+for alternative splicing event estimation. Chen et al.
+[@tag:Chen2015_trans_species] used deep belief networks to learn
+phosphorylation states of a common set of signaling proteins in primary
+cultured bronchial cells collected from rats and humans treated
+with distinct stimuli. Interpreting species as different modalities
+representing similar high-level concepts, they showed that DBNs were able
+to capture cross-species representation of signaling mechanisms in response
+to a common stimuli. Another application used DBNs for joint unsupervised
+feature learning from cancer datasets contained gene expression,
+DNA methylation, and miRNA expression data [@tag:Liang2015_exprs_cancer].
+This approach allowed for the capture of intrinsic relationships in
+different modalities and for better clustering performance over conventional
+k-means based methods.
+
+Deep graphical models such as DBNs are considered to be well suited for
+multimodal learning tasks since they are aimed to learn joint probability
+distribution from inputs. However, it is difficult task that requires
+specialized algorithms for training and sampling.  While it is possible
+to fine-tune these models for discriminatory tasks, they are typically
+outperformed by commonly used convolutional neural networks (CNN)
+trained with backpropagation. Multimodal learning with CNNs is usually
+implemented as a collection of individual networks, in which each learns
+representations from single data type. These individual representations are
+further concatenated before or within fully-connected layers. FIDDLE
+[@tag:Eser2016_fiddle] is an example of multimodal CNNs that represents an
+ensemble of individual networks that take as inputs a number of genomic
+datasets, including NET-seq, MNase-seq, ChIP-seq, RNA-seq, and
+raw DNA sequence to predict Transcription Start Site-seq (TSS-seq) outputs.
+The combined model radically improves performance over separately trained
+datatype-specific networks, suggesting that it learns the synergistic
+relationship between datasets.
```

Comment:
I agree. Also tagging @ueser in case there is a specific result to highlight with FIDDLE that best highlights the improved performance.